src/tools/tidy/src/ext_tool_checks.rs

```diff
@@ -73,6 +153,8 @@ fn check_impl(
     let python_fmt = lint_args.contains(&"py:fmt") || python_all;
     let shell_all = lint_args.contains(&"shell");
     let shell_lint = lint_args.contains(&"shell:lint") || shell_all;
+    let clang_all = lint_args.contains(&"clang");
+    let clang_fmt = lint_args.contains(&"clang:fmt") || clang_all;
```

Comment:
I think that this should be named e.g. `cpp` and `cpp:fmt`? We are formatting C++ code, same as above we're linting/formatting Python/bash code. The fact that it is done with `clang-format` is an implementation detail (same as that we use e.g. `ruff`).

Reply:
Yes, it's better.